src/main/java/seedu/duke/parsers/ModHappyParser.java

```diff
@@ -69,6 +71,8 @@ private Parser getCommandParser(String commandWord) throws UnknownCommandExcepti
             return new DeleteParser();
         case (MARK_COMMAND_WORD):
             return new MarkParser();
+        case (SAVE_COMMAND_WORD):
+            return new NoArgumentParser(commandWord);
```

Comment:
Let's combine this with the fallthrough cases in lines 64-65.

Reply:
okk